.github/workflows/ci.yml

```diff
@@ -37,7 +37,29 @@ jobs:
       - name: Build package
         run: |
           python scripts/build/install.py
```

Comment:
Here, just do, `CXXFLAGS=--coverage CFLAGS=--coverage python scripts/build/install.py`.

Reply:
The above should work.

Reply:
made the changes in ci.yml file